Source/OxyPlot/Axes/Rendering/HorizontalAndVerticalAxisRenderer.cs

```diff
@@ -274,6 +274,40 @@ protected virtual void RenderAxisTitle(Axis axis, double titlePosition)
 
             var lpt = this.GetAxisTitlePositionAndAlignment(axis, titlePosition, ref angle, ref halign, ref valign);
 
+            var titleSize = this.RenderContext.MeasureText(axis.ActualTitle, axis.ActualTitleFont, axis.TitleFontSize, axis.TitleFontWeight);
+
+            ScreenPoint areaBottomLeft = new ScreenPoint(0, 0);
+            ScreenPoint areaTopRight = new ScreenPoint(0, 0);
+
+            switch (axis.Position)
+            {
+                case AxisPosition.Left:
+                    areaTopRight = new ScreenPoint(lpt.X, lpt.Y + titleSize.Width * 0.5);
+                    areaBottomLeft = new ScreenPoint(lpt.X + titleSize.Height, lpt.Y - titleSize.Width * 0.5);
+                    break;
+                case AxisPosition.Right:
+                    areaTopRight = new ScreenPoint(lpt.X - titleSize.Height, lpt.Y + titleSize.Width * 0.5);
+                    areaBottomLeft = new ScreenPoint(lpt.X, lpt.Y - titleSize.Width * 0.5);
+                    break;
+                case AxisPosition.Top:
+                    areaTopRight = new ScreenPoint(lpt.X - titleSize.Width * 0.5, lpt.Y + titleSize.Height);
+                    areaBottomLeft = new ScreenPoint(lpt.X + titleSize.Width * 0.5, lpt.Y);
+                    break;
+                case AxisPosition.Bottom:
+                    areaTopRight = new ScreenPoint(lpt.x - titleSize.Width * 0.5, lpt.Y);
+                    areaBottomLeft = new ScreenPoint(lpt.X + titleSize.Width * 0.5, lpt.Y - titleSize.Height);
+                    break;
+
+            }
+            // TODO: ensure min area of 5px x X to capture clicks, o this above acc. to axis position
```

Comment:
Did this TODO get resolved?

Reply:
@ChrisCC6 this and comment code below could do with being addressed. If you don't have time to work on this let us know and I'll try to take a look into this and @Jonarw 's questions because I'm keen for this PR to go through.

Reply:
@VisualMelon I added two examples based on @Jonarw's code above and added the AxisArea to be set for the MagnitudeAxisRenderer.
The TODO is resolved as well.

Reply:
Thanks indeed! Shall try to take a proper look myself later today.